File: templates/Uwp/_composition/MVVMLight/Project.SplitView/Behaviors/NavigationViewHeaderBehavior_postaction.cs
//{[{
using Param_RootNamespace.ViewModels;
//}]}

namespace Param_RootNamespace.Behaviors
{
    public class NavigationViewHeaderBehavior : Behavior<WinUI.NavigationView>
    {
        protected override void OnAttached()
        {
            //^^
            //{[{
            ViewModelLocator.Current.NavigationService.Navigated += OnNavigated;
            //}]}
        }

        protected override void OnDetaching()
        {
            //^^
            //{[{
            ViewModelLocator.Current.NavigationService.Navigated -= OnNavigated;
            //}]}
        }
    }
}

File: templates/Uwp/_composition/MVVMLight/Project.SplitView._VB/Behaviors/NavigationViewHeaderBehavior_postaction.vb
'{[{
Imports Param_RootNamespace.ViewModels
'}]}

Namespace Behaviors
    Public Class NavigationViewHeaderBehavior
        Inherits Behavior(Of WinUI.NavigationView)

        Protected Overrides Sub OnAttached()
            '^^
            '{[{
            AddHandler ViewModelLocator.Current.NavigationService.Navigated, AddressOf OnNavigated
            '}]}
        End Sub
    End Class
End Namespace